### PR TITLE
Remove scala test kit

### PR DESCRIPTION
**Pull request by dpetrick**

### CODE REVIEW

File: .envrc [hunk 1]
```diff
@@ -1,41 +1,27 @@
...
-
-# Controls scala test kit binary build mode ("debug", "release"): Defaults to "release" if not set.
-export TEST_BINARY_BUILD_MODE=debug
 
-# Local dev env
+### Various local dev env vars ###
 export PRISMA_CONFIG_PATH=$(pwd)/prisma.yml
 export PRISMA_DML_PATH=$(pwd)/datamodel_v2.prisma
-
 export PRISMA2_BINARY_PATH="/usr/local/lib/node_modules/prisma2/"
```
Comment (tomhoule) [inline .envrc:10]: I don't use this file usually so I don't have any strong opinion, but I'm curious, what is this variable for?

File: .envrc [hunk 2]
```diff
@@ -1,41 +1,27 @@
...
-
-# Controls Scala test kit test complexity ("simple", "complex")
-export TEST_MODE=simple
-
-# Controls scala test kit binary build mode ("debug", "release"): Defaults to "release" if not set.
-export TEST_BINARY_BUILD_MODE=debug
 
-# Local dev env
+### Various local dev env vars ###
 export PRISMA_CONFIG_PATH=$(pwd)/prisma.yml
```
Comment (tomhoule) [inline .envrc:8]: Is this still used?

File: .envrc (hunk 1 above)
Comment (dpetrick) [reply .envrc:10]: This is something that the makefile uses 🤷

File: .envrc (hunk 2 above)
Comment (dpetrick) [reply .envrc:8]: Actually no, this can be removed.

**Pull request merged**